### ISSUE TITLE
Forward message if whitelisted word came after message got edited in source chat

### ISSUE THREAD

**Issue by rijeshkumar** [aahnik/tgcf#134]
Bot not forwarding a message if whitelist word came after a message got edited on source chat. It is only forwarding that message if it contained a whitelist word was there before it got edited. (In live mode)

**Comment by aahnik** [comment]
yes, this is expected behavior. nothing can be done to solve it.

if the whitelisted word is not present in the original message, then `tgcf` will not deal with it.

**Comment by rijeshkumar** [comment]
Can't you make the bot to wait for some seconds (say 30 seconds) and check again if whitelist word is there if a message doesn't have whitelist word at first?
